src/components/home/Landing.jsx

```diff
@@ -1,5 +1,53 @@
+import Image from "next/image";
+import logo from "@/public/prishi-logo.webp";
+import landing from "@/public/landing.webp";
+import Button from "@/components/Button";
+import Link from "next/link";
+
 const Landing = () => {
-  return <div>Landing</div>;
+  return (
+    <div className="flex h-screen w-screen">
+      <div className="flex flex-col w-8/12 justify-center gap-3">
+        <div className="flex justify-center">
+          <Image src={logo} alt="rishi-logo" className="w-24 h-auto" />
+        </div>
+
+        <div className="flex justify-center">
```

Comment:
you don't need to wrap the "Project RISHI at UCR" container in this extra flex container. delete this extra flex container and just add text-center to the Project RISHI at UCR container